### ISSUE TITLE
Add SDG background color classes: 2-5, 8-11, 13, 16, 17

### ISSUE THREAD

**Issue by JessicaLucindaCheng** [hackforla/website#4344]
### Dependency
- #4233 

### Overview
We need to add background color classes using the SDG (Sustainable Development Goal) color variables to our repo so we can use these classes with their corresponding, matching SDG icons.

### Action Items
- [x] In the `_sass/elements`, create a new file: `_background-sdg-colors.scss`
- [x] In the file you just created, add the following:
```
/**
 * Sustainable Development Goal (SDG) Background Colors
 *
 * These background color classes match the colors of the SDG icons provided by the United Nations (UN):
 * https://www.un.org/sustainabledevelopment/news/communications-material/
 */

.background-color-sdg2 { background-color: $color-sdg2; }
.background-color-sdg3 { background-color: $color-sdg3; }
.background-color-sdg4 { background-color: $color-sdg4; }
.background-color-sdg5 { background-color: $color-sdg5; }
.background-color-sdg8 { background-color: $color-sdg8; }
.background-color-sdg9 { background-color: $color-sdg9; }
.background-color-sdg10 { background-color: $color-sdg10; }
.background-color-sdg11 { background-color: $color-sdg11; }
.background-color-sdg13 { background-color: $color-sdg13; }
.background-color-sdg16 { background-color: $color-sdg16; }
.background-color-sdg17 { background-color: $color-sdg17; }
``` 
- [x] In the `_sass/main.scss`, under the `Elements` section, add this:
```
@import 'elements/background-sdg-colors';
```

#### Merge Team, Technical Lead, or Product Manager
- [x] Once the pull request associated with this issue is merged, go to each of the following issues and check if all the issues/dependencies listed under the `Dependency` section are closed or checked off. If the dependencies are all closed or checked, move those issues into the `New Issue Approval` column, remove the `Dependency` label on it, and add a `ready for dev lead` label to it.
  - [x] #4187 

### Resources/Instructions
- [_sass/elements](https://github.com/hackforla/website/tree/gh-pages/_sass/elements)
- [_sass/main.scss](https://github.com/hackforla/website/blob/gh-pages/_sass/main.scss)
- [SDGs_HEX_Colors tab in HfLA: Projects: SDG and Targets Google Sheet](https://docs.google.com/spreadsheets/d/1lV8d8AMSJ5HOiTWlFE7lzbqiKxqMmh8pHADcJMdp7JE/edit#gid=741671421)
- This issue is tracked in #4153

**Comment by JessicaLucindaCheng** [comment]
Once this issue is in the New Issue Approval column, a dev lead needs to review the issue and determine if it is ready for a PM to prioritize it. If it isn't ready, a dev lead will need to edit this issue so that it is ready for a PM to prioritize it.

**Comment by qngyn** [comment]
Progress: Already done and push
Blockers: None

**Comment by jdingeman** [comment]
@qngyn - you said that this issue has been done and pushed, but I do not see a linked PR that will close this issue? Did you make a pull request to incorporate the changes you made in your local into gh-pages?

**Comment by jdingeman** [comment]
@qngyn - I noticed that you're already assigned to an issue. Please wait for your other issue, #3998, to be merged and closed before doing another issue. I am going to unassign you from this issue.

**Comment by wKategianes** [comment]
Availability for this week: 12pm-5pm PST
My estimated ETA for completing this issue: 4/20/23

**Comment by jdingeman** [comment]
Hi @wKategianes - when you take on an issue, please remember to move the issue into the In Progress column. Thanks!

**Comment by roslynwythe** [comment]
@wKategianes  This issue has been flagged for review by the dev lead team because there has been no activity or comments in the past 2 weeks.   Please leave a comment outlining your progress, availability and ETA, and describe any blockers or problems that you are experiencing, then remove the "2 weeks inactive" label.  Thank you!

**Comment by ExperimentsInHonesty** [comment]
@wKategianes We have not heard from you this issue is time sensitive, so I am removing you from this issue.

**Comment by jenny-alexander** [comment]
Availability: Friday May 19 10-12pm.
ETA: Friday May 19 10-12pm.

**Comment by jenny-alexander** [comment]
Review ETA: 05/19/23 10-12pm
Availability: 05/19/23 10-2pm
